test/servico/Cst20Test.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package servico;
+
+import auxiliar.Perfil;
+import datamapper.PopulateDB;
+import dominio.Usuario;
+import java.util.List;
+import modelo.UsuarioModel;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['import' should be separated from previous statement.](https://www.codacy.com/app/calebepb/substituicao/file/2734087542/issues/source?bid=3308597&fileBranchId=3351034#l15)